src/mcp/client/streamable_http.py

```diff
@@ -415,12 +415,12 @@ async def _handle_sse_response(
                 await ctx.read_stream_writer.send(e)
 
         # Auto-reconnect if stream ended without completion and we have priming event
-        if not is_complete and has_priming_event and last_event_id:
```

Comment:
For the life of me I couldn't get these code paths covered - I'm not sure why, as the example client and server demonstrate this works and the conformance tests pass too:

<img width="1679" height="755" alt="CleanShot 2025-11-24 at 22 05 09" src="https://github.com/user-attachments/assets/1c61b13a-f45e-4704-86ff-704707e68c82" />

Will try again in a follow-up but would prefer not to block on this...